arbitrum-docs/run-arbitrum-node/more-types/02-run-validator-node.mdx

```diff
@@ -1,76 +1,150 @@
-  ```shell
-  docker run --rm -it  -v /some/local/dir/arbitrum:/home/user/.arbitrum @latestNitroNodeImage@ --parent-chain.connection.url=https://l1-mainnet-node:8545 --chain.id=42161 --node.staker.enable --node.staker.parent-chain-wallet.only-create-key --node.staker.parent-chain-wallet.password="SOME SECURE PASSWORD"
-  ```
-- Wallet file will be created under the mounted directory inside the `arb1/wallet/` directory for Arb1, or `nova/wallet/` directory for Nova. Be sure to backup the wallet, it will be the only way to withdraw stake when desired
-
-### Running an allowlisted defensive validator
-
-- A defensive validator requires that a wallet has already been created using the above steps
-- Defensive validator wallets do not need to be funded initially
-- If a defensive validator detects a deviation, it will log `bringing defensive validator online because of incorrect assertion`, and wait for funds to be added to wallet so stake can be posted and a dispute created
-- Here is an example of how to run an allowlisted defensive validator for Arbitrum One:
-  ```shell
-  docker run --rm -it  -v /some/local/dir/arbitrum:/home/user/.arbitrum @latestNitroNodeImage@ --parent-chain.connection.url=https://l1-mainnet-node:8545 --chain.id=42161 --node.staker.enable --node.staker.strategy=Defensive --node.staker.parent-chain-wallet.password="SOME SECURE PASSWORD"
-  ```
-- For Orbit chains, you need to set the `--chain.info-json=<Orbit Chain's chain info>` flag instead of `--chain.id=<chain id>`
-- To verify validator is working, this log line shows the wallet is setup correctly:
-  ```shell
-  INFO [09-28|18:43:49.367] running as validator                     txSender=0x... actingAsWallet=0x... whitelisted=true strategy=Defensive
-  ```
-  - `whitelisted` should be `true` after your wallet has been added to the allowlist
-  - `strategy` should be `Defensive`
-  - `txSender` and `actingAsWallet` should both be present and not `nil`
-  - The log line `validation succeeded` shows that the L2 block validator is working
-  - The log line `found correct assertion` shows that the L1 validator is working
-
-#### Orbit chains: grant whitlelist
-
-- You need to be the chain owner to include a new validator address in the allowlist:
-- Find your `upgradeExecutor` contract address.
-- Send transactions to the `executeCall` method of the`upgradeExecutor` contract and set the `target` address to your Rollup contract's address, set the `targetCalldata` to `0xa3ffb772{Your new allowlist validator address}`. (`0xa3ffb772` is the signature of `setValidator(address[],bool[])`)
-- Call your Rollup contract's `isValidator(address)` and check the result.
+Validators are nodes that choose to participate in the rollup protocol to advance the state of the chain securely. Since the activation of <a data-quicklook-from="bold">BoLD</a>, chains can now choose to make validation permissionless. You can learn more about BoLD [here](/how-arbitrum-works/bold/gentle-introduction.mdx).
```

Comment:
```suggestion
Validators are nodes that choose to participate in the rollup protocol to advance the state of the chain securely. Since the activation of <a data-quicklook-from="bold">BoLD</a>, chains can now choose to make validation permissionless. You can learn more in the [BoLD introduction](/how-arbitrum-works/bold/gentle-introduction.mdx).
```